scripts/beck/st/error_injection.py

```diff
@@ -164,128 +176,156 @@ def ReadMetaData(offset):
 	with open(filename, "rb") as mdata:
 		mdata.seek(offset)
 		data=binascii.hexlify((mdata.read(8))[::-1])
-		print("** Metadata at offset {} is {}".format(hex(offset), data))
+		logger.info("** Metadata at offset {} is {}".format(hex(offset), data))
```

Comment:
file mdata is not closed?

Reply:
If a file handler open using with then no need to close it. It will automatically be closed once its definition/function gets over.